pages/common/podman-login.md

```diff
@@ -0,0 +1,16 @@
+# podman login
+
+> Log in to a container registry.
+> More information: <https://docs.podman.io/en/latest/markdown/podman-login.1.html>.
+
+- Log in to a registry (non-persistent on Linux; persistent on Windows/macOS):
+
+`podman login {{registry.example.org}}`
+
+- Log in to a registry persistently on Linux:
+
+`podman login --authfile=$HOME/.config/containers/auth.json {{registry.example.org}}`
+
+- Log in to an insecure (HTTP) registry:
+
+`podman login --tls-verify=false {{registry.example.org}}`
```

Comment:
The main podman page contains options without `=`, so this should be possible, right?
https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#option-syntax
> Prefer using a space instead of the equals sign (`=`) to separate options from their arguments
```suggestion
`podman login {{registry.example.org}}`

- Log in to a registry persistently on Linux:

`podman login --authfile $HOME/.config/containers/auth.json {{registry.example.org}}`

- Log in to an insecure (HTTP) registry:

`podman login --tls-verify false {{registry.example.org}}`
```